ArenaControl/Stage3.cpp

```diff
@@ -170,7 +170,10 @@ void Stage3::step(uint32_t timestamp)
       
       /* did we just transition into center on this step? */
       if (!prevCenter) {
-         turns = (encoder + (ONE_REVOLUTION/2)) / ONE_REVOLUTION;
```

Comment:
Good catch - I incorporated this change with a slight modification (use ?: operator, but same logic)